client/src/components/ActionMenu/ActionMenu.js

```diff
@@ -31,6 +31,8 @@ class ActionMenu extends PureComponent {
       dropdownMenuProps,
       dropdownToggleClassNames,
       className,
+      // remove props that we don't want to pass through to Dropdown
+      toggleCallback, // eslint-disable-line no-unused-vars
```

Comment:
Why not just `delete restProps.toggleCallback`? It's already a clone, so there's no risk of breaking references.